docs/deployment/edge.md

```diff
@@ -64,13 +64,9 @@ chmod +x install-edge.sh
 sudo -E ./install-edge.sh
 ```
 
-*For an offline installation*
+!!! Note "Offline installation"
 
-Copy the OCI bundle and the install-edge.sh script to the remote server.
-
-Then mirror the various images from docker-compose.yaml into your private registry, and update the references from i.e. `image: ghcr.io/openfaasltd/gateway` to the equivalents in your registry.
-
-If your system is unable to install apt, yum, or pacman packages, due to limited network access, then set the `SKIP_OS` environment to 1. The list of packages is available in the `install_required_packages` section of the script.
```

Comment:
This note should be retained in the new location.

Reply:
I removed the note because we don't use the `install-edge.sh` script for air-gapped installations. All the script does is install packages, download and unpack the installation oci image and run the install.sh script within.

Both the download step for packages and the installation oci can not run without internet connection.  Users can just run the `install.sh` script from the installation bundle directly.